ibis/backends/pandas/__init__.py

```diff
@@ -148,6 +150,53 @@ def create_table(self, table_name, obj=None, schema=None):
         if schema is not None:
             self.schemas[table_name] = schema
 
+    def parquet_files(self, files, schema=None):
+        """
+        Create a table backed by Parquet files on disk.
+
+        Parameters
+        ----------
+        files : list of str
+            A list of file paths.
+
+        schema : Schema, optional
+            The schema of the data. If not given, it will be inferred
+            from the *first* file listed.
+        """
+        if not files:
+            raise ValueError("Must pass as least one file")
+
+        # TODO: it'd be nice to support things like directories and
+        # globs here (would probably require being able to assume
+        # PyArrow)
+
+        items = []
+        for raw_path in files:
+            # Assume filesystem paths for now; in the future we could
+            # describe S3 paths, etc.
+            path = Path(raw_path).expanduser().resolve()
+            if not path.is_file():
+                raise ValueError(
+                    f"'{raw_path}' does not exist or is not a file!"
+                )
+
+            if not schema:
+                # It'd be nice to be able to assume PyArrow here to avoid
+                # reading the entire file
```

Comment:
If you add `pyarrow` as a dependency of the `pandas` extra in `pyproject.toml` then you can make this assumption.